weave

```diff
@@ -102,13 +103,15 @@ check_docker_version() {
         exit 1
     fi
 
-    # guard against https://github.com/docker/docker/issues/8632
-    if [ "$DOCKER_VERSION" = "1.3.0" ] ; then
-        echo "You are running Docker version $DOCKER_VERSION, which contains a bug that prevents weave from working properly. Please upgrade." >&2
+    FLAG=$(awk 'BEGIN { print ("$DOCKER_VERSION" < "$MIN_DOCKER_VERSION") ? "1" : "0" }')
```

Comment:
This indeed fails for version numbers of the form x.y.z. I will try to come up with something that works better.